<!-- -->
<launch>

  <arg name="p" default="false"/>
  <arg name="s" default="false"/>
  <node pkg="cooperation_setup" type="connection_client_setup" name="connection_client_subscriber" output="screen" if="$(arg s)">
    <param name="client_mode" value="subscriber"/>
  </node>
  <node pkg="cooperation_setup" type="connection_client_setup" name="connection_client_publisher" output="screen" if="$(arg p)">
    <param name="client_mode" value="publisher"/>
  </node>

</launch>
